### PR TITLE
Add invalidation to /project when project is updated

### PR DESCRIPTION
**Pull request by BeckyPollard**
## Summary of changes:
_(see attached image for changes preview)_

- Add invalidation for `/project` when a project updates. Should show updates now when getting projects from collection.

| (BEFORE) After changing project client | (AFTER) After changing project client |
| ----- | ------ |
| Screen Shot 2022-10-19 at 4 43 12 PM | Screen Shot 2022-10-19 at 4 43 16 PM |

Prettier changed more than just adding the invalidaiton link, but that's all the changes really. Only noticed after push.

----
Feedback, nitpicks, and/or corrections encouraged. 🤖

### CODE REVIEW

File: src/project/controller/item.ts [hunk 1]
```diff
@@ -19,30 +19,34 @@ class Project extends Controller {
...
 
     project.name = body.name;
     await projectService.update(project);
 
     ctx.status = 204;
-
+    ctx.response.links.add({
+      rel: 'invalidates',
+      href: '/project',
+    });
```
Comment (evert) [inline src/project/controller/item.ts:49]: Just curious, why was this needed?

Comment (BeckyPollard) [reply src/project/controller/item.ts:49]: Great question!
When updating a project (like updating the associated client) the `/project` list of projects collection wouldn't show the updated client, it would just be blank,
I tried adding the option `const {items} = useCollection(props.resource, {refreshOnStale: true});` to the CollectionLinks, where we're getting the list of projects, but it didn't resolve the issue.
The only ways I could remember to get resources/collection to update properly when another is changed is to pass the option `refreshOnStale`, or to add a link to whatever request to invalidate whatever is dependant on that updated info. Since refreshOnStale didn't work, I went with the invalidation links, which worked out. 🤷

Comment (evert) [reply src/project/controller/item.ts:49]: Does this have to do with fetching the title of the project? This makes sense to me if we fetch a list of projects (but not the children) so titles are just extracted from the `item` link.

**Review by evert — APPROVED** [review]

File: src/project/controller/item.ts (hunk 1 above)
Comment (BeckyPollard) [reply src/project/controller/item.ts:49]: Yeah! You got it!
